traits/src/lib.rs

```diff
@@ -144,3 +146,15 @@ pub trait DataProvider<Key, Value> {
 pub trait PriceProvider<CurrencyId, Price> {
 	fn get_price(base: CurrencyId, quote: CurrencyId) -> Option<Price>;
 }
+
+pub trait CombineData<Key, TimestampedValue, Moment> {
+	type expiresIn: Get<Moment>;
+	type minimumCount: Get<u32>;
```

Comment:
those are implementation details, shouldn't be part of trait. they should be part of the  `struct DefaultCombineData` (or whatever it calls)